graphing/grapher.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import argparse
+parser = argparse.ArgumentParser(description="Create graphs in the current directory")
+parser.add_argument("outfile")
+parser.add_argument("--remote", action="store_true", help="Use this option on a remote"
+                                                          "ssh machine.")
+args = parser.parse_args()
+import matplotlib
+if args.remote:
+    matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+import glob
+import re
+import numpy as np
+if __name__ == "__main__":
+
+    fnames = glob.glob("lookupresults.*")
+    assert len(fnames) != 0, "Couldn't find file of the name lookupresults.* in current directory"
+    conditions = set()
+    for f in fnames:
+        m = re.search(r".*?\.(.*)", f)
+        conditions.add(m.group(1))
+
+    for condition in conditions:
+        guesses = []
+        with open("lookupresults.{}".format(condition)) as f:
+            for line in f:
+                try:
+                    guess_num = int(line.split("\t")[5])
+                    if guess_num < 0:
+                        continue
+                    guesses.append(guess_num)
+                except:
```

Comment:
What exception is being ignored here? Without knowing why this is here, I might be wrong, but if it is because the split fails or the conversion to int fails, then I think we shouldn't ignore that. If the user input is bad, we don't want to silently ignore the errors. If this for something else, could you please add a comment explaining why?